src/main/java/org/auscope/portal/server/web/controllers/SF0BoreholeController.java

```diff
@@ -110,11 +110,11 @@ public void doFilterStyle(
         //		}
 
         String filter = this.boreholeService.getFilter(boreholeName,
-                custodian, dateOfDrilling, maxFeatures, bbox,
+                custodian, dateOfDrilling, dateOfDrilling, maxFeatures, bbox,
```

Comment:
Is my screen being funny with me or can you confirm what I am seeing is an error

custodian, dateOfDrilling, dateOfDrilling, maxFeatures, bbox

why 2 dateOfDrilling?